azure-pipelines.yml

```diff
@@ -65,51 +65,6 @@ jobs:
       artifactName: coverage
   timeoutInMinutes: 30
 
-- job: Linux_Mono
-  pool:
-    vmImage: ubuntu-20.04
-  steps:
-  - task: UseDotNet@2
-    displayName: Stick with .NET Core SDK version compatible to Mono toolchain
-    inputs:
-      packageType: sdk
-      version: 3.1.101
-      installationPath: $(Agent.ToolsDirectory)/dotnet
-  - template: .azure-pipelines/mono.yml
-    parameters:
-      configuration: $(configuration)
-      turnServerUrls: $(turnServerUrls)
-      testArguments: -parallel none -stoponfail -verbose
-  timeoutInMinutes: 30
-
-- job: macOS_Mono
-  pool:
-    vmImage: macOS-10.15
-  steps:
-  - template: .azure-pipelines/mono.yml
```

Comment:
Is this template file still used?  If no more used, please remove it as well.

Reply:
macOS_Unity uses it.